content/wasm-functions/using-cron-trigger.md

```diff
@@ -32,16 +34,158 @@ Use Cases for Cron Jobs:
 
-## Managing Cron Jobs
+```toml
+[[trigger.http]]
+route = "/hello"
+component = "hello-cron-job"
+```
+
+Let's modify the response body so we can validate that our application is running on a specific interval. To do so, navigate to `src/index.js` and add the following snippet of code:
+
+<!-- @selectiveCpy -->
+
+```JavaScript
+// For AutoRouter documentation refer to https://itty.dev/itty-router/routers/autorouter
+import { AutoRouter } from 'itty-router';
+
+let router = AutoRouter();
+
+// Route ordering matters, the first route that matches will be used
+// Any route that does not return will be treated as a middleware
+// Any unmatched route will return a 404
+router
+    .get("/hello", () => {
+    // Grab datetime
+    const now = new Date().toISOString();
+    // Return response with time of invocation
+    return new Response(`Hello from a cron component at ${now}`, { status: 200 });
+});
+
+
+addEventListener('fetch', (event) => {
+    event.respondWith(router.fetch(event.request));
+});
+```
+
+To test locally, run the following command:
+
+<!-- @selectiveCpy -->
+
+```console
+spin build --up
+```
+
+If it is working as expected, you should see a message returned along the lines of (but with an accurate datetime):
+
+<!-- @selectiveCpy -->
+
+```console
+curl localhost:3000
+```
+
+<!-- @nocpy -->
+
+```console
+Hello from a cron component at 2025-03-31T21:45:25.562Z
+```
+
+## Deploy Spin Application to Fermyon Wasm Functions
+
+Before applying our cron trigger, we must have our Spin application running on the Fermyon Wasm Functions platform. Deploy the application with:
+
+<!-- @selectiveCpy -->
+
+```console
+spin aka deploy
+```
+
+Upon successful deployment, you should receive a domain name that you can use to test the application:
+
+<!-- @nocpy -->
+
+```console
+curl https://ec8f3c23-b73d-4d51-aa69-aabd7ddc132a.aka.fermyon.tech/hello
+Hello from a cron component at 2025-04-02T03:16:14.212Z% 
+```
+
+# Creating a Cron Job for Your Spin Application
+
+Now we will use the `spin aka crons` command to invoke the Spin app's HTTP endpoint at our desired interval. The syntax for this request follows standard crontab format.
+
+> `spin aka cron` supports single digits in each cron expression field and intervals in every position except the day of the week (e.g., `* */12 * * *` for every 12 hours). The smallest supported cron configuration is every 5 minutes (`*/5 * * * *`). Other syntax elements like comma-separated lists and ranges are not supported. Please feel free to[leave feedback](https://fibsu0jcu2g.typeform.com/to/G2u4tPcP) if your desired range is not supported. 
+
+Let's create a cron job that runs every hour. Since scheduling is based on UTC, you may need to [convert your local time to UTC](https://www.worldtimebuddy.com/?pl=1&lid=100&h=100&hf=1) to ensure correct execution.
+
+<!-- @selectiveCpy -->
+
+```console
+spin aka crons create "0 * * * *" "/hello" "hello-cron-job"
```

Comment:
Maybe explain what these fields are, since they are not named in the command.  I'm guessing the second is the route to be hit.  Is the third the name of the app to be hit?  Or the name of the job?  (In which case how does it know which app to hit?  Is this an "app in current directory" thing?)

If the third parameter is _not_ the name of the app, it would be good to choose a different string from the name of the app, so as to avoid ambiguity.